rust/htp/src/connection_parser.rs

```diff
@@ -278,7 +278,7 @@ pub struct ConnectionParser {
     /// The logger structure associated with this parser
     pub(crate) logger: Logger,
     /// A reference to the current parser configuration structure.
-    pub(crate) cfg: Rc<Config>,
+    pub(crate) cfg: Rc<&'static Config>,
```

Comment:
From the commit message:

>  and trust C caller to take care of it

Is the `Rc` really doing anything here?  Can you fully trust it?  Or would a *const Config be just a safe?

Reply:
It was doing something...

> Or would a *const Config be just a safe?

May be better.

I will read https://doc.rust-lang.org/std/primitive.pointer.html